hosting.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: default
+---
+
+First of all, thanks for considering hosting a bookclub meeting! Your willingness to share your enthusiasm and perspective for a book is exactly why we started this group!
+
+Here are some things you'll want to think about:
+* TOC
```

Comment:
I had thought about this, but I think it's important enough to be stated in full on each page, so that's how I'm leaving it for now -- I thought about writing a short version and then including the longer link, but I couldn't decide what to leave in or take out. Couldn't get to a better version than essentially "don't be a jerk, read more here..."